tensorflow/contrib/tensor_forest/kernels/v4/decision_node_evaluator.h

```diff
@@ -96,7 +96,7 @@ std::unique_ptr<DecisionNodeEvaluator> CreateDecisionNodeEvaluator(
 std::unique_ptr<DecisionNodeEvaluator> CreateBinaryDecisionNodeEvaluator(
     const decision_trees::BinaryNode& node, int32 left, int32 right);
 
-struct CandidateEvalatorCollection {
+struct CandidateEvaluatorCollection {
```

Comment:
is there a good way to check this?  it does not appear anywhere in the primary codebase.

Reply:
Can we rollback this change from this PR as this is a breaking change.

We can push the other documentation changes for now and deal with this independently.